### ISSUE TITLE
mintinstall v. 8.0.8 points to wrong flatpak URL.

### ISSUE THREAD

**Issue by QuintonMcLeod** [linuxmint/mintinstall#326]
```
 * Cinnamon version (cinnamon --version)
   - Please specify if you are using the daily builds PPA
     (https://launchpad.net/~linuxmint-daily-build-team/+archive/ubuntu/daily-builds).
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
 * Graphics hardware *and* driver used
 * 32 or 64 bit
 * Attach ~/.xsession-errors, or /var/log/syslog
 ```

**Issue**
mintinstall, which is part of mint-meta-cinnamon, points to: https://flathub.org/repo/flathub.flatpakrepo, which is incorrect. The URL should be https://dl.flathub.org/repo/flathub.flatpakrepo. Since it's pointing to the wrong URL, the following error appears:

error: Can't load uri https://flathub.org/repo/flathub.flatpakrepo: Unacceptable TLS certificate

**Steps to reproduce**
Simply attempt to install mint-meta-cinnamon within a Ubuntu 20.04 environment.

**Expected behaviour**
The package should just install.

**Other information**

**Configuration info, if applicable**
If this bug report is related to an Applet, Desklet or Extension, please paste (or use a pastebin service) the offending extension's settings, if it has any.  You can obtain this by opening its configuration, clicking the 'hamburger' button in the upper-right corner of the window, and selecting "Export to a file". **Please be sure to review the contents and remove any personal data it may contain.**

```

Please paste here

```

**Comment by clefebvre** [comment]
Fixed in mintinstall 8.0.9

**Comment by clefebvre** [comment]
Fix backported also in 8.0.3.1, 7.9.7.2 and 7.9.5.2.
